src/composables/pollAccountInfo.ts

```diff
@@ -0,0 +1,45 @@
+import { IDefaultComposableOptions } from '@/types';
+import { useFungibleTokens } from './fungibleTokens';
+
+let isInitialized = false;
+let pollAvailableTokens: NodeJS.Timer;
+let pollTokenBalances: NodeJS.Timer;
+let pollDefaultNames: NodeJS.Timer;
+
+export function usePollAccountInfo({ store }: IDefaultComposableOptions) {
+  const {
+    loadTokenBalances,
+    loadAvailableTokens,
+  } = useFungibleTokens();
+
+  function stopPolling() {
+    clearInterval(pollDefaultNames);
+    clearInterval(pollAvailableTokens);
+    clearInterval(pollTokenBalances);
+  }
+
+  function startPolling() {
+    pollDefaultNames = setInterval(() => store.dispatch('names/setDefaults'), 10000);
+    pollAvailableTokens = setInterval(() => loadAvailableTokens(), 60000);
+    pollTokenBalances = setInterval(() => loadTokenBalances(), 10000);
+  }
+
+  function handleVisibilityChange(event: Event) {
```

Comment:
I think you could get rid of this metod in favour of `createPollingBasedOnMountedComponents`

Reply:
Totally